### PR TITLE
update to group_by

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr updates the the exports code example to 'group_by' and not 'group_bys'. it also update the saved queries parameter too.  reported by Steve Dowling here: https://dbt-labs.slack.com/archives/C02NCQ9483C/p1707782794575719

### CODE REVIEW

File: website/docs/docs/use-dbt-semantic-layer/exports.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ description: "Use exports to materialize tables to the data platform on a schedu
 sidebar_label: "Materialize with exports"
 ---
 
-# Materialize queries with exports <Lifecycle status='new'/>
+### Materialize queries with exports <Lifecycle status={[ 'team', 'enterprise' ]}/>
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/exports.md]:
```suggestion
# Materialize queries with exports <Lifecycle status= 'new' />
```

Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/exports.md]:
```suggestion
### Materialize queries with exports <Lifecycle status='new' />
```

File: website/docs/docs/use-dbt-semantic-layer/exports.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ description: "Use exports to materialize tables to the data platform on a schedu
 sidebar_label: "Materialize with exports"
 ---
 
-# Materialize queries with exports <Lifecycle status='new'/>
+### Materialize queries with exports <Lifecycle status='new' />
```
Comment (nghi-ly) [inline website/docs/docs/use-dbt-semantic-layer/exports.md]: sugg: i've been taught that it's bad practice to skip heading levels (ie use them in order)

```suggestion
# Materialize queries with exports <Lifecycle status='new' />
```

**Review by nghi-ly — COMMENTED** [review]
looks great! left one sugg

File: website/docs/docs/use-dbt-semantic-layer/exports.md (hunk 2 above)
Comment (mirnawong1) [reply website/docs/docs/use-dbt-semantic-layer/exports.md]: oh dear, this was a mistake! i was testing the issue you had with the lifecycle in the separate pr and also had a failed deploy. but i forgot to remove the h3 🙈 

thank you so much for the catch!

**Comment by mirnawong1** [comment]
hey @nghi-ly ! thanks can you rereview pls

**Review by nghi-ly — APPROVED** [review]

**Pull request merged**